code/app/src/main/java/com/example/snackoverflow/Ingredient.java

```diff
@@ -27,6 +27,14 @@ public Ingredient(String description, String dt, String location, int amount, in
         this.category = category;
     }
 
+    // For Recipe
+    public Ingredient(String description, Integer amount, Integer unit, String category) {
```

Comment:
@raquibk @Bhavnoor-Kaur double check this pls

Reply:
This constructor is missing attribute. Why do we need another constructor? And I noticed, this is the old version of the file! @Hitarth-Kothari Please update your branch with master before making changes!